.github/workflows/contracts-build.yml

```diff
@@ -0,0 +1,78 @@
+name: Contracts-Build
+
+on:
+  push: 
+    branches: [ main, dev ]
+    tags: ['*']
+  pull_request:
+
+env:
+  cwd: ${{github.workspace}}/packages/contracts
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+
+  unit-test-and-lint:
+    runs-on: ubuntu-20.04
```

Comment:
need ^21.04

Reply:
Can you elaborate more? Test suite of contract package runs well on any linux distribution and macOS.

Reply:
right now we're mocking out some of the steps, once proofs are actually being generated we need 21.04 distro to run the binaries that generate proofs

Reply:
Ack, I will setup our own self-hosted runner (CI instance) on AWS and will make integration tests run on that instance; GitHub-hosted runner didn't support 21.04.

Reply:
Ok let's just open this up as a new issue